src/sparkx/Oscar.py

```diff
@@ -387,7 +388,9 @@ def print_particle_lists_to_file(self, output_file: str) -> None:
                 raise ValueError("The number of output per event is empty.")
             if self.num_events_ is None:
                 raise ValueError("The number of events is empty.")
-            if self.num_events_ > 1:
+            if self.num_events_ == 0:
+                warnings.warn("The number of events is zero.")
```

Comment:
I think we should be consistent between Jetscape, where you have an error, and Oscar, where you warn. I think a warning is the more reasonable choice. Such things can happen on purpose.